Code/VideoPlayerKit.m

```diff
@@ -60,20 +61,21 @@ - (void)setTopView:(UIView *)topView
                                                           object:self
                                                            queue:[NSOperationQueue mainQueue]
                                                       usingBlock:^(NSNotification *note) {
-                                                          [UIView animateWithDuration:0.0f
+                                                          [UIView animateWithDuration:0.4f
```

Comment:
it would be good if you add a property to customize this duration but have 0.4f as default

Reply:
Ah okay yea this makes sense to match the status bar fade. Maybe just a constant to say statusBarDuration so the 0.4 doesn't seem like a magic number.

Reply:
I created a controlsAnimationDuration constant with a comment explaining that the 0.4 value matches the status bar duration. Let me know what you think!